content/docs/guides/components/tftraining.md

```diff
@@ -165,6 +165,14 @@ VERSION=v0.2-branch
 ks registry add kubeflow-git github.com/kubeflow/kubeflow/tree/${VERSION}/kubeflow
 ks pkg install kubeflow-git/examples
 
+# list available tf-job prototypes and choose right one match with your crd
```

Comment:
I think it will be clearer if we end the code snippet above this line, then make this line straight text, so it looks like this:


Create the component (update version as appropriate).
 	 
 ```
 CNN_JOB_NAME=mycnnjob
 VERSION=v0.2-branch
 	 
ks registry add kubeflow-git github.com/kubeflow/kubeflow/tree/${VERSION}/kubeflow
ks pkg install kubeflow-git/examples
```

Choose a tf-job prototype from the following list of available prototypes, to match the CRD you're using:

* `io.ksonnet.pkg.tf-job-operator` -  A TensorFlow job operator.
* `io.ksonnet.pkg.tf-job-simple` -  A simple TFJob to run CNN benchmark
* `io.ksonnet.pkg.tf-job-simple-v1alpha1` -  A simple TFJob to run CNN benchmark
* `io.ksonnet.pkg.tf-job-simple-v1beta1` - A simple TFJob to run CNN benchmark

Run the `generate` command:

```
ks generate tf-job-simple ${CNN_JOB_NAME} --name=${CNN_JOB_NAME}
```

Reply:
Cool. Agree